backend/src/main/java/com/google/rolecall/jsonobjects/UserInfo.java

```diff
@@ -114,31 +128,30 @@ public static Builder newBuilder() {
     @JsonProperty("email")
     public abstract Builder setEmail(String email);
 
+    @JsonProperty("phoneNumber")
+    public abstract Builder setPhoneNumber(String phoneNumber);
+
     @JsonProperty("dateJoined")
     @JsonFormat(pattern="MM-dd-yyyy")
     public abstract Builder setDateJoined(Calendar dateJoined);
 
-    @JsonProperty("emergencyContcatName")
-    public abstract Builder setEmergencyContactName(String emergencyContcatName);
+    // Roles
+    @JsonProperty("isAdmin")
+    public abstract Builder setIsAdmin(Boolean isAdmin);
 
-    @JsonProperty("emergencyContactName")
-    public abstract Builder setEmergencyContactNumber(String emergencyContactNumber);
-
-    @JsonProperty("comments")
-    public abstract Builder setComments(String comments);
+    @JsonProperty("isCoreographer")
+    public abstract Builder setIsCoreographer(Boolean isCoreographer);
 
-    @JsonProperty("role")
-    public abstract Builder setRole(String role);
+    @JsonProperty("isDancer")
+    public abstract Builder setIsDancer(Boolean isDancer);
 
-    @JsonProperty("isActive")
-    public abstract Builder setIsActive(Boolean isActive);
+    @JsonProperty("isOther")
+    public abstract Builder setIsOther(Boolean isOther);
 
+    // Permissions
```

Comment:
Yes, it would be a significant change. Also, given how "under designed" this project is, having Roles and Permissions un-nested gives us more flexibility to get the right permissions "is spite" of whatever roles are present.